### PR TITLE
Add device function collection#125

### LINKED ISSUE #125: Implement NetworkDeviceFunctionCollection
As oneview-redfish-user, I access "/redfish/v1/Systems/<server_hardware_uuid>/NetworkAdapters/<deviceId>/NetworkDeviceFunctions" and receive the collection of NetworkDeviceFunctions objects associated with this adapter, one for each 3-tuple (physical_port, virtual_port, port_function). The ID should be the concatenation of each element id separated by "_".

### PR DESCRIPTION
**Pull request by ffdarkpenguin**
closes #125

### CODE REVIEW

File: oneview_redfish_toolkit/api/network_device_function_collection.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        self.redfish["@odata.type"] = \
+            "#NetworkDeviceFunctionCollection.NetworkDeviceFunctionCollection"
+        self.redfish["Name"] = "Network Device Function Collection"
+        members_count = 0
+        self.redfish["Members"] = list()
+        index = int(device_id) - 1
+        devices = server_hardware["portMap"]["deviceSlots"][index]
+        for port in devices["physicalPorts"]:
+            physical_port = str(port["portNumber"])
+            for virtual_port in port["virtualPorts"]:
```
Comment (danielreed) [inline oneview_redfish_toolkit/api/network_device_function_collection.py]: You might check with Ricardo but I can't remember if we were going to look at virtual ports.

File: redfish.conf [hunk 2]
```diff
@@ -13,7 +13,7 @@ userName = administrator
 password =
 
 [ssl]
-SSLType = adhoc
+SSLType = disabled
```
Comment (danielreed) [inline redfish.conf]: Did you mean to change this? I prefer disabled.

Comment (ricardoas) [reply redfish.conf]: I prefer too... but not on this patch! It's not NetworkFunction related.

File: oneview_redfish_toolkit/api/network_device_function_collection.py (hunk 1 above)
Comment (ricardoas) [reply oneview_redfish_toolkit/api/network_device_function_collection.py]: @danielreed I though our final agreement was that the NetworkDeviceFunction was a one to one mapping with each virtual port... is it something else?

File: redfish.conf (hunk 2 above)
Comment (ffdarkpenguin) [reply redfish.conf]: Yes I meant to. It's really a pain have to accept the cert every time during development. Ok, we change it in another patch

**Pull request merged**